.eslintignore

```diff
@@ -1,4 +1,5 @@
 package-lock.json
 package.json
 tailwind.config.js
-postcss.config.js
\ No newline at end of file
+postcss.config.js
+/tests/
```

Comment:
why are you ignoring your tests for lining...?

Reply:
changed to include now